curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/633928c6b0baf16c0b0d823b.md

```diff
@@ -0,0 +1,98 @@
+---
+id: 633928c6b0baf16c0b0d823b
+title: Step 41
+challengeType: 0
+dashedName: step-41
+---
+
+# --description--
+
+You have created two radio buttons. However, if you submit the form, it does not require you to check any of them.
+
+This is not good, since you need to know what type of account the user requires.
+
+To correct this add a `required` attribute to `personal-account or business-account`.
+
+# --hints--
+
+You should use the `required` attribute for the `input` tag.
+
+```js
+assert.equal(document.getElementsByName("account-type").required, true);
+```
```

Comment:
this test will fail, you need to update the seed code of all following steps

Reply:
So I should add this test to all subsequent steps and remove it from this one?

Reply:
No, the test stay here. But you need to update the seed code of all the following steps and the final solution